<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>plugin::datasource::iris::CControlWidget</class>
 <widget class="QWidget" name="plugin::datasource::iris::CControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QSlider" name="horizontalSlider_minMag">
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="value">
        <number>60</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_minMag">
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>9.900000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>6.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_maxRadius">
       <property name="text">
        <string>Max radius</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="spinBox_maxRadius">
       <property name="suffix">
        <string> km</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>20038</number>
       </property>
       <property name="value">
        <number>20038</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_minMag">
       <property name="text">
        <string>Min magnitude</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="horizontalSlider_maxRadius">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>20038</number>
       </property>
       <property name="value">
        <number>20038</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_time">
       <property name="text">
        <string>Time interval</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit_from">
       <property name="dateTime">
        <datetime>
         <hour>18</hour>
         <minute>0</minute>
         <second>0</second>
         <year>1999</year>
         <month>12</month>
         <day>30</day>
        </datetime>
       </property>
       <property name="displayFormat">
        <string>yyyy.MM.dd</string>
       </property>
       <property name="timeSpec">
        <enum>Qt::UTC</enum>
       </property>
       <property name="date">
        <date>
         <year>1999</year>
         <month>12</month>
         <day>30</day>
        </date>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit_to">
       <property name="dateTime">
        <datetime>
         <hour>17</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2013</year>
         <month>12</month>
         <day>30</day>
        </datetime>
       </property>
       <property name="time">
        <time>
         <hour>17</hour>
         <minute>0</minute>
         <second>0</second>
        </time>
       </property>
       <property name="displayFormat">
        <string>yyyy.MM.dd</string>
       </property>
       <property name="timeSpec">
        <enum>Qt::UTC</enum>
       </property>
       <property name="date">
        <date>
         <year>2013</year>
         <month>12</month>
         <day>30</day>
        </date>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_find">
       <property name="text">
        <string>Find</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="treeView">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QLabel" name="label_secondAfter">
       <property name="text">
        <string>Second after</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="horizontalSlider_secondAfter">
       <property name="minimum">
        <number>60</number>
       </property>
       <property name="maximum">
        <number>43200</number>
       </property>
       <property name="value">
        <number>7200</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="spinBox_secondAfter">
       <property name="minimum">
        <number>60</number>
       </property>
       <property name="maximum">
        <number>43200</number>
       </property>
       <property name="value">
        <number>7200</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_secondBefore">
       <property name="text">
        <string>Second before</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="horizontalSlider_secondBefore">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>3600</number>
       </property>
       <property name="value">
        <number>300</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="spinBox_secondBefore">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>3600</number>
       </property>
       <property name="value">
        <number>300</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>0</number>
       </property>
       <property name="format">
        <string>Downlaoding %p%</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_OpenOrCancel">
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>horizontalSlider_minMag</tabstop>
  <tabstop>doubleSpinBox_minMag</tabstop>
  <tabstop>horizontalSlider_maxRadius</tabstop>
  <tabstop>spinBox_maxRadius</tabstop>
  <tabstop>dateEdit_from</tabstop>
  <tabstop>dateEdit_to</tabstop>
  <tabstop>pushButton_find</tabstop>
  <tabstop>treeView</tabstop>
  <tabstop>horizontalSlider_secondBefore</tabstop>
  <tabstop>spinBox_secondBefore</tabstop>
  <tabstop>horizontalSlider_secondAfter</tabstop>
  <tabstop>spinBox_secondAfter</tabstop>
  <tabstop>pushButton_OpenOrCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider_secondAfter</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_secondAfter</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>565</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>564</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_secondAfter</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_secondAfter</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>554</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>559</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_secondBefore</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_secondBefore</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>532</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>533</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_secondBefore</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_secondBefore</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>531</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>527</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
